src/ni/vsbuild/steps/StepFactory.groovy

```diff
@@ -36,6 +36,14 @@ class StepFactory implements Serializable {
          return new LvSetConditionalSymbolStep(script, mapStep, lvVersion)
       }
 
+      if(type == 'lvUTF') {
+         return new LvUnitTestFrameworkStep(script, mapStep, lvVersion)
+      }
+	  
+	  if(type == 'lvVITester') {
```

Comment:
You have a mix of tabs and spaces in the file. Make sure you are only using spaces.

Reply:
I think I completed this